docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -1173,6 +1186,362 @@ SQL> POPULATE INDEX index_target_table4 ON target_table4;
+Task:  POPULATE INDEX  Status: Ended      Time: 2018-01-18 13:35:12.461
+Task:  POPULATE INDEX  Status: Ended      Elapsed Time:    00:00:07.589
+--- SQL operation complete.
+
+Start Time             2018/01/18 13:34:37.053647
+End Time               2018/01/18 13:35:12.893891
+Elapsed Time                      00:00:35.840244
+Compile Time                      00:00:00.435855
+Execution Time                    00:00:35.402620
+
+SQL>LOAD WITH REBUILD INDEXES INTO target_table2 SELECT * FROM source_table WHERE id > 300;
+UTIL_OUTPUT
+---------------------------------------------------------------------------------
+Task:  LOAD            Status: Started    Object: TRAFODION.SEABASE.TARGET_TABLE2                                                     
+Task:  CLEANUP         Status: Started    Time: 2018-01-18 13:35:25.480                                                         
+Task:  CLEANUP         Status: Ended      Time: 2018-01-18 13:35:25.493
+Task:  CLEANUP         Status: Ended      Elapsed Time:    00:00:00.013
+Task:  DISABLE INDEXE  Status: Started    Time: 2018-01-18 13:35:25.493                                                         
+Task:  DISABLE INDEXE  Status: Ended      Time: 2018-01-18 13:35:38.844
+Task:  DISABLE INDEXE  Status: Ended      Elapsed Time:    00:00:12.591
+Task:  LOADING DATA    Status: Started    Time: 2018-01-18 13:35:38.845                                                         
+       Rows Processed: 999700 
+       Error Rows:     0 
+Task:  LOADING DATA    Status: Ended      Time: 2018-01-18 13:35:43.491
+Task:  LOADING DATA    Status: Ended      Elapsed Time:    00:00:05.407
+Task:  COMPLETION      Status: Started    Time: 2018-01-18 13:35:43.491                                                         
+       Rows Loaded:    999700 
+Task:  COMPLETION      Status: Ended      Time: 2018-01-18 13:35:45.920
+Task:  COMPLETION      Status: Ended      Elapsed Time:    00:00:01.601
+Task:  POPULATE INDEX  Status: Started    Time: 2018-01-18 13:35:45.920                                                         
+Task:  POPULATE INDEX  Status: Ended      Time: 2018-01-18 13:35:56.322
+Task:  POPULATE INDEX  Status: Ended      Elapsed Time:    00:00:11.230
+--- SQL operation complete.
+
+Start Time             2018/01/18 13:35:24.693410
+End Time               2018/01/18 13:35:56.754441
+Elapsed Time                      00:00:32.061031
+Compile Time                      00:00:00.449236
+Execution Time                    00:00:31.611112
+
+SQL>SET PARSERFLAGS 1;
+--- SQL operation complete.
+
+SQL>SELECT COUNT(*) FROM TABLE(INDEX_TABLE index_target_table2);
+(EXPR)
+--------------------
+             1000000
+
+--- 1 row(s) selected.
+```
+
+* This example compares the execution time of using LOAD Statement without options and 
+using `LOAD WITH REBUILD INDEXES` when the CQD `TRAF_LOAD_ALLOW_RISKY_INDEX_MAINTENANCE` 
+is turned *ON*. The former takes shorter time than the latter.

```

Comment:
"The former takes less time than the latter" is a bit better.

Reply:
Thanks Dave, your comment has been incorporated :)